phpunit/class-rest-shortcodes-controller-test.php

```diff
@@ -99,10 +99,8 @@ public function data_get_item_with_field_combinations() {
 					'shortcode' => 'Any Random Text',
 				),
 				array(
-					'html'  => 'Any Random Text',
-					'type'  => 'html',
-					'style' => '',
-					'js'    => '',
+					'html' => '<p>Any Random Text</p>',
```

Comment:
I slightly worry about this one...

I was thinking 

```
                if ( $output === wpautop( $shortcode ) ) {
			$data['html'] = __( 'Looks like this isn\'t rich content', 'gutenberg' );
			return rest_ensure_response( $data );
		}
```

I'm just not sure if that's right either.